linkml/generators/graphqlgen.py

```diff
@@ -50,6 +50,13 @@ def visit_class_slot(self, cls: ClassDefinition, aliased_slot_name: str, slot: S
             slotrange = slotrange + "!"
         return f"\n    {lcamelcase(aliased_slot_name)}: {slotrange}"
 
+    def visit_enum(self, enum: EnumDefinition):
+        out = f"enum {enum.name}\n  {{"
+        for permissible_value in enum.permissible_values:
+            out = "\n    ".join([out, permissible_value])
+        out = "\n".join([out, "  }\n\n"])
+        return out
```

Comment:
Using Jinja templates might be meaningful, but I only wanted to add support for enumerations without making deeper changes in the generator. I would prefer to get enumerations supported before opening such a construction site as the one Jinja would mean.

Otherwise I like your proposal, so accepting it.

Reply:
Of course :) makes sense